File: 1-Iniciante/ex1177.js
// entrada
let input = require('fs').readFileSync('/dev/stdin', 'utf-8'); // adicione uma barra "/" antes do "dev" para ser aceito no beecrowd
let n = parseInt(input);

// processamento & saida
let vetor = [];
let valor = 0;

for(let i = 0, cont = 0; cont < 1000; i++, cont++){
    if(i === n) i = 0;
    console.log(`N[${cont}] = ${i}`);
}

File: 1-Iniciante/ex1177.py
#entrada
t = int(input())

#saida
n = []
j = 0
for i in range(0, 1000):
    if j == t:
        j = 0
    n.append(j)
    print(f'N[{i}] = {j}')
    j += 1
